docs/journalist.rst

```diff
@@ -469,45 +464,59 @@ choose **Wipe** to delete them.
 
 .. _`guide to removing metadata`: https://freedom.press/training/everything-you-wanted-know-about-media-metadata-were-afraid-ask/
 
+.. _malware_risks:
+
 Risks From Malware
 ~~~~~~~~~~~~~~~~~~
 
-As long as you are using the latest version of Tails, you should be
-able to open submitted documents with a low risk of malicious
-files compromising the *Secure Viewing Station*. However, even if a
-compromise does occur, Tails is designed so that the next time you
-reboot, the malware will be gone.
-
-It is crucial, however, that you have a strategy for dealing with malware before
-you move documents off the *Secure Viewing Station* in electronic form.
-SecureDrop does not scan for or automatically remove malware. If you copy an
-original file you received via SecureDrop to your everyday workstation, and that
-file contains malware, you may still end up running the malware on your everyday
-workstation.
-
-For this reason, we recommend taking additional precautions. Printing a file
-is often safer than exchanging it electronically, and it has the additional
-benefits of removing embedded metadata (except for printer codes, watermarks,
-or similar identifiers that may not be visible to the naked eye).
-
-Alternatively, you can use the tools provided within Tails to examine
-documents, or convert files from one format to another (e.g., export a Word
-document as PDF). Fully mitigating the risks of malware is beyond the scope
-of this documentation.
+As long as you are using the latest version of Tails on your *Secure Viewing Station*,
+you can open submitted documents with a low risk of malicious files compromising
+it. Tails only preserves changes to your persistent volume across reboots,
+making it difficult for malware to take hold. The air-gap prevents malware
+from "phoning home". Keeping your *Secure Viewing Station* up-to-date is the
+single best way to protect its security.
+
+If you copy files you have received via SecureDrop to your everyday workstation
+and open them there in their original form, the risks from malware are
+significantly greater. SecureDrop does not scan for or remove malware, and your
+everyday workstation likely has a much greater `attack surface <https://en.wikipedia.org/wiki/Attack_surface>`__
+than the *Secure Viewing Station*.
```

Comment:
While this is also considered attack surface, the "phoning home" aspect might be important to reiterate here.

Reply:
Done in 46ec379e91862a74a08b814ab1f51265170f6477.